docs/graphql/core/remote-schemas/adding-schema.rst

```diff
@@ -42,7 +42,7 @@ To merge your remote schema with the GraphQL engine's auto-generated schema:
 
     Head to the ``Remote Schemas`` tab of the console and click on the ``Add`` button.
 
-    .. thumbnail:: /img/graphql/core/business-logic/add-remote-schemas-interface.png
+    .. thumbnail:: /img/graphql/core/remote-schemas/add-remote-schemas-interface.png
```

Comment:
@marionschleifer is it that you changed the location of this particular image and not deleted it? :) Just to be sure.

Reply:
Yes, exactly 🙂 this is because we changed the structure of docs.